portfolio/src/main/webapp/script.js

```diff
@@ -28,24 +28,26 @@ function addRandomGreeting() {
 }
 
 function displayHobbies() {
-    const title = document.getElementById('title');
+    const title = document.getElementById('content-title');
     title.innerText = "This is a list of my main hobbies:";
     $(function(){
-        $("#panel").load("main_page_clickables/hobbies.html"); 
+        $("#content-panel").load("main_page_clickables/hobbies.html"); 
     });
 }
 
 function displayAbout() {
-    const title = document.getElementById('title');
+    const title = document.getElementById('content-title');
     title.innerText = "Here is some background information about me:";
     $(function(){
-        $("#panel").load("main_page_clickables/aboutme.html"); 
+        $("#content-panel").load("main_page_clickables/aboutme.html"); 
     });
 }
 
 function activateListItem() {
-    $('#hobbiesList a').on('click', function (e) {
-        e.preventDefault()
-        $(this).tab('show')
-    });
+    $(document).ready(function() { 
```

Comment:
You don't need the activateListItem() function to do this. You can call `$(document).ready(..)` directly from script.js, and jquery will run the callback as soon as the page has finished loading.